src/WatchDogs.FakeSource/TradeModel.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Bogus;
+
+namespace WatchDogs.FakeSource;
+
+public record TradeModel
```

Comment:
Record is suitable for immutable types which suits us because we don't need (and want) to change Trade objects.

Reply:
ok.